src/index.js

```diff
@@ -2,6 +2,7 @@ import EventEmitter from 'events';
 
 import { setPromisesDependency, getPromisesDependency } from './utils';
 import { main, readyFreddy, getEnv, getValue, setValue, apifyClient, call } from './actor';
+import { launchPuppeteer } from './puppeteer';
```

Comment:
Because puppeteer is optional, we need to use require() inside of launchPuppeteer function. We can throw some user-friendly error to tell them they probably chose a wrong image.